ramanujantools/pcf/pcf_test.py

```diff
@@ -136,3 +136,17 @@ def test_blind_delta_sequence_agrees_with_blind_delta():
         expected_deltas.append(pcf.delta(dep, limit))
 
     assert expected_deltas == actual_values
+
+
+def test_as_latex():
```

Comment:
Split this test to more specific cases
Test all of the functionality: the start point, the depth and generic/non-generic.